.travis.yml

```diff
@@ -1,5 +1,11 @@
 language: rust
 
+branches:
+  only:
+  - master
+  - /^v\d+\.\d+(\.\d+)?(-\S*)?$/
+  - /.*-release/
```

Comment:
We can even use a shorter `/-release/` regex. Ruby matches it anywhere in the string. This form allows releases like `0.10.0-release-rc1`. But maybe we want `/-release$/` instead to allow it only at the end.

Though, this is not critical and PR can be merged as it is now.